data/crawler/conf/spark-env.sh

```
@@ -0,0 +1,80 @@
+LD_PRELOAD=/opt/bitnami/common/lib/libnss_wrapper.so
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)